internal/endpoints/endpointsFetcher.go

```diff
@@ -0,0 +1,60 @@
+package endpoints
```

Comment:
Can you send me what you found that says to put this in `internal`? It seems weird to me. 

I wonder if more broadly the organization should look like 

* main.go
  * only the entry execution code.
* endpointquerier.go
  * all the querying code that's in main.go
  * maybe also the endpoint fetcher code? Or maybe not because it's specific to just the json file that you created for these initial endpoints... but the model itself should be general and generally available.

Reply:
https://github.com/golang-standards/project-layout/tree/master/internal